source/features/cleanup-repo-filelist-actions.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import PlusIcon from 'octicon/plus.svg';
+import SearchIcon from 'octicon/search.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {groupButtons} from '../github-helpers/group-buttons';
+
+function init(): void {
+	const searchButton = select('.btn[data-hotkey="t"]')!;
+	searchButton.classList.add('tooltipped', 'tooltipped-ne');
+	searchButton.classList.remove('mr-2');
+	searchButton.setAttribute('aria-label', 'Go to file');
+	searchButton.firstChild!.replaceWith(<SearchIcon/>);
+
+	const addButtonWrapper = searchButton.nextElementSibling!;
+	addButtonWrapper.classList.add('tooltipped', 'tooltipped-ne');
+	addButtonWrapper.setAttribute('aria-label', 'Add file');
+	const addButton = select('.dropdown-caret', addButtonWrapper)!.parentElement!;
+	addButton.classList.add('d-md-block');
+	addButton.classList.remove('d-md-flex', 'ml-2');
+	addButton.textContent = '';
+	addButton.append(<PlusIcon/>);
+
+	groupButtons([searchButton, addButtonWrapper]);
+
+	const downloadButton = select('get-repo details');
+	if (downloadButton) {
+		downloadButton.classList.add('tooltipped', 'tooltipped-ne');
+		downloadButton.setAttribute('aria-label', 'Download');
```

Comment:
I guess this is GitHub's own "Code" button, not the "Download" button added by RGH?

```suggestion
		downloadButton.setAttribute('aria-label', 'Code');
```

Reply:
I typed it wrong

But a single word "Code" is confusing, maybe "Download code" or something

Reply:
It's what GitHub uses, so I don't think it's too bad. "Download" doesn't fit IMO, because you can also clone the repo from that dialog.

Reply:
Good!